backend/src/main/resources/static/semantic-hub-openapi.yaml

```diff
@@ -128,6 +128,49 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /models/bulk:
+    post:
+      tags:
```

Comment:
You could a description to make the design decision clear, that the endpoint will return all models that are available in the Semantic Hub even if some of the requested models from the SemanticModelList in the request body are not available.